src/components/Editor/HighlightLine.js

```diff
@@ -122,6 +122,21 @@ export class HighlightLine extends Component<Props> {
 
     const doc = getDocument(sourceId);
     doc.addLineClass(editorLine, "line", "highlight-line");
+    this.resetHighlightLine(doc, editorLine);
+  }
+
+  resetHighlightLine(doc, editorLine) {
```

Comment:
Looks like CI is failing and wants Flow types here.

```
yarn run v1.9.4
$ flow
Launching Flow server for /root/debugger.html
Spawned flow server (pid=3900)
Logs will go to /tmp/flow/zSrootzSdebugger.html.log
Monitor logs will go to /tmp/flow/zSrootzSdebugger.html.monitor_log
Error -------------------------------------------------------------------- src/components/Editor/HighlightLine.js:128:22

Missing type annotation for `doc`.

   128|   resetHighlightLine(doc, editorLine) {
                             ^^^


Error -------------------------------------------------------------------- src/components/Editor/HighlightLine.js:128:27

Missing type annotation for `editorLine`.

   128|   resetHighlightLine(doc, editorLine) {
                                  ^^^^^^^^^^


Error -------------------------------------------------------------------- src/components/Editor/HighlightLine.js:129:36

Cannot call `getComputedStyle` with `document.querySelector(...)` bound to `elt` because null [1] is incompatible with
`Element` [2].

   src/components/Editor/HighlightLine.js:129:36
    129|     const style = getComputedStyle(document.querySelector(".editor-wrapper"));
                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

References:
   /tmp/flow/flowlib_23c0bb84/dom.js:952:50
    952|   querySelector(selector: string): HTMLElement | null;
                                                          ^^^^ [1]
   /tmp/flow/flowlib_23c0bb84/dom.js:3787:40
   3787| declare function getComputedStyle(elt: Element, pseudoElt?: string): any;
                                                ^^^^^^^ [2]



Found 3 errors
error Command failed with exit code 2.```

Reply:
@wldcordeiro Can you help me out with `Cannot call getComputedStyle with document.querySelector(...) bound to elt because null [1] is incompatible with
Element [2].`